### DISCUSSION TITLE
Type aliases

### DISCUSSION THREAD

**Discussion by gaku-sei** [mint-lang/mint#131] [Language Development]
It could be nice to have some type aliases, with the simple form:

```
alias EmptyPromise = Promise(Never, Void)
```

or maybe

```
alias EmptyPromise : Promise(Never, Void)
```

Allowed in providers/components/store/etc... and at the top level probably.

If generics land one day in Mint, it might become extra useful 😄

**Comment by gdotdesign** [comment]
I don't know about this, it's not for 1.0 for sure. Let's discuss it after that :smile:
